# This patch needs to contain the version number as referenced in toolkit/scripts/NPToolkit.version
--- a/NPToolkit/Config/Version.pm
+++ b/NPToolkit/Config/Version.pm
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-our $VERSION = 3.3;
+our $VERSION = "3.5rc1";
 
 =head1 NAME
 
@@ -99,16 +99,7 @@
     my ( $self, @params ) = @_;
     my $parameters = validate( @params, {} );
 
-    my $version;
-
-    if ( open( OUTPUT, "-|", $self->{NPTOOLKIT_VERSION_BIN} ) ) {
-        $version = <OUTPUT>;
-        close( OUTPUT );
-
-        chomp( $version ) if ( $version );
-    }
-
-    return $version;
+    return $VERSION;
 }
 
 =head2 save_state()
